tests/support/fixtures.ts

```diff
@@ -365,7 +365,12 @@ export async function createCobsFixture(peer: RadiclePeer) {
     "Add README\n\nThis commit adds more information to the README",
     "feature/add-readme",
     () => Fs.writeFile(Path.join(projectFolder, "README.md"), "# Cobs Repo"),
-    ["Let's add a README", "This repo needed a README"],
+    [
+      "Let's add a README",
+      "This repo needed a README",
+      "Let's also check that <RID> and <OID> get rendered correctly",
+      "Outside of a code block and `inside <RID> and <OID> of a codeblock`",
```

Comment:
We should test closing tags, and trusted tags as well, though maybe not in the fixture. We should have a dedicated test for the markdown rendering of tags.

Reply:
yeah we can do that

Reply:
Added a new test in the markdown repository to the visual tests that verifies the closing tags and trusted tags.

<img width="779" alt="Bildschirmfoto 2023-06-19 um 11 47 48" src="https://github.com/radicle-dev/radicle-interface/assets/7912302/224be1f9-1b34-4cd1-9e43-cbd272b8aef4">